### ISSUE TITLE
Portuguese trained data fails to recognize @

### ISSUE THREAD

**Issue by tigermarques** [tesseract-ocr/tessdata_best#51]
Hi,

I'm using tesseract to perform OCR in a document that contains an email address. Using the eng trained data, it recognizes the email address correctly (but naturally fails in all accented characters). When I switched to the por trained data, it picks up all portuguese characters correctly, but fails to recognize the @ in email addresses.
Is there any additional configuration needed for this special characters?

Thank you!

**Comment by tfmorris** [comment]
#54 describes a similar problem for Turkish. It's likely that the training data for the two languages didn't include any examples of email addresses.

**Comment by stweil** [comment]
That's correct. See [por.unicharset](https://github.com/tesseract-ocr/langdata_lstm/blob/main/por/por.unicharset) and [tur.unicharset](https://github.com/tesseract-ocr/langdata_lstm/blob/main/tur/tur.unicharset) which do not contain the `@` character.  So that character was not part of the training data and will therefore never be recognized by those models.

I suggest to use `-l Latin` in similar cases.
